rewrite-java-21/src/test/java/org/openrewrite/java/MethodTypeTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.Issue;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.test.RewriteTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.openrewrite.java.Assertions.java;
+
+class MethodTypeTest implements RewriteTest {
+    @Issue("https://github.com/openrewrite/rewrite/issues/5400")
+    @Test
+    void guavaMethodParameterNamesPresent() {
+        rewriteRun(
+          spec -> spec.parser(JavaParser.fromJavaVersion().classpath("guava")),
```

Comment:
I only wanted this test and runtime dependency added to one of the parsers, instead of all of them, hence why this test is here and not in rewrite-java-test/tck.

Since it's such a small change we can also remove this test here completely and rely on a downstream test that uses the parameter names in
- https://github.com/openrewrite/rewrite/pull/5923